.vscode/launch.json

```diff
@@ -268,6 +268,23 @@
 				"group": "2_attach"
 			}
 		},
+		{
+			"type": "pwa-chrome",
+			"request": "attach",
+			"name": "Attach to VS Code",
```

Comment:
You might has a more descriptive task name related to ADS, such as "Attach to render process", etc.  Though I thought we already had tasks for all the processes.

Reply:
Can you do this in a separate PR to avoid multiple unrelated changes being made in a single PR?